examples/navier-stokes/navierstokes.c

```diff
@@ -514,20 +514,20 @@ PetscErrorCode SetUpDM(DM dm, problemData *problem, const char *prefix,
     ierr = DMCreateDS(dm); CHKERRQ(ierr);
     /* Wall boundary conditions are zero velocity and zero flux for density and energy */
     ierr = DMAddBoundary(dm, DM_BC_ESSENTIAL, "wall", "Face Sets", 0,
-                         3,(PetscInt[]) {1,2,3},
-                         (void(*)(void))problem->bc, bc->nwall, bc->walls, ctxSetup);
+    3,(PetscInt[]) {1,2,3},
+    (void(*)(void))problem->bc, bc->nwall, bc->walls, ctxSetup);
```

Comment:
Is this what you want? I don't know any editors or style guides that behave this way. Sometimes I think we should abandon astyle due to its inconsistencies. clang-format is reliable, but draconian, especially with respect to error checking macros.

Reply:
Yeah, I didn't like that either. I fixed it.

I'm happy with using astyle or something else, but whatever we do, I think we need to make `make style` part of the CI. Right now it gets left off frequently and then one of us tries to check the style of new code and gets hit with a bevy of old changes from another PR because we don't always `make style` our PRs.